zoo-css-challenge/style.css

```
@@ -114,16 +140,28 @@ footer {
 
 .tigers {
   background-color: #0b3c49;
-  color: #f0e100;
+  color:#FFA500;
+}
+
+.tigers h2 {
+  color: #b17402;
 }
 
 .giraffe {
-  background-color: #465775;
-  color: #56e39f;
+  background-color:#483C46;
+  color: #BEEE62;
 }
 
 .giraffe h2 {
-  color: #ef6f6c;
+  color: #F4743B;
+}
+
+.giraffe h3 {
```

```suggestion
.giraffe h2,
.giraffe h3 {
```
As these two selectors have the same CSS declaration, we can combine them like so